invenio_theme/templates/semantic-ui/invenio_theme/page.html

```diff
@@ -26,8 +26,7 @@
     {%- endblock head_meta %}
 
     {%- block head_title %}
-      {%- set title = title or _(config.THEME_SITENAME) or _('Invenio') %}
-      <title>{{title}}</title>
+      <title>{%- if title %}{{title}} | {% endif -%}{{ _(config.THEME_SITENAME) or _('Invenio') }}</title>
```

Comment:
LGTM code-wise.
all the instances running invenio will be affected by this change, was that the goal ?

Reply:
Yep, this is beneficial for all instances in my opinion

The only problem I can foresee is if they haven't set `THEME_SITENAME` then they will have Invenio appended

Reply:
could you add a section "Update configuration variables" under 
https://github.com/inveniosoftware/docs-invenio-rdm/pull/607/files#diff-39da3e2af9ef047f00739348879d5a91714f7b1cf3f693c959a9e6154c2b9795R125
explaining exactly this? It should go into the upgrade guidelines for v12

Reply:
Added a comment! btw this variable has already been contributed to cookiecutter :) https://github.com/inveniosoftware/cookiecutter-invenio-rdm/pull/273

Reply:
Ok to merge from me if there is not a chance we'll have ``Zenodo | Zenodo``?  For that we'll just need to check if ``title`` is every set to ``config.THEME_SITENAME``